ui/appui-layout-react/src/appui-layout-react/widget-panels/Panel.tsx

```diff
@@ -233,36 +143,19 @@ export function WidgetPanel() {
         className={className}
         ref={ref}
         style={style}
-        onTransitionEnd={handleTransitionEnd}
+        onTransitionEnd={(e) => {
+          if (e.target !== elementRef.current)
+            return;
+
+          handleTransitionEnd();
```

Comment:
Shouldnt that condition be in `handleTransitionEnd` itself as it gets the same `ref` anyway ?